.gitignore

```diff
@@ -23,3 +23,4 @@ karmada-controller-manager
 karmada-scheduler
 karmada-webhook
 karmadactl
+karmada-replica-estimator
```

Comment:
This is the RPC `server` name, right?
What's the client name look like?

Reply:
Ah😄, we don't need an additional client binary because the client is karmada-scheduler.

Reply:
right, what a stupid question!